code.js

```diff
@@ -1,3 +1,39 @@
 function mergesort(array) {
+    if (array.length <= 1) {
+        return array;
+    }
+
+    let array2 = new Array(array.length);
```

Comment:
Not in place.

Reply:
Ok my new stuff should be in place. I believe my run time analysis is still correct.